### PR TITLE
Add a workaround script for arm64 tch-rs build issue (#180)

### PR DESCRIPTION
**Pull request by antimora**

### CODE REVIEW

File: burn-tch/build.rs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+use std::env;
+fn main() {
```
Comment (nathanielsimard) [inline burn-tch/build.rs:2]: This is so small, but I would like spaces between imports and functions 😅. I guess the `build.rs` is not formatted by `cargo fmt` ?

File: burn-tch/build.rs [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    // Temporary workaround for https://github.com/burn-rs/burn/issues/180
+    // Remove this once tch-rs upgrades to Torch 2.0 at least
+
+    if cfg!(all(target_arch = "aarch64", target_os = "macos")) {
+        let message = "Run scripts/fix-tch-build-arm64.py to fix environment variables for torch";
```
Comment (nathanielsimard) [inline burn-tch/build.rs:7]: I would add the link to the issue in the comment, so it's clearer why this step is necessary.

Comment (nathanielsimard) [reply burn-tch/build.rs:7]: And I guess most poeple using burn from crates.io won't have access to the script, so pointing to the link online is important.

**Review by nathanielsimard — CHANGES_REQUESTED** [review]

File: burn-tch/build.rs (hunk 1 above)
Comment (antimora) [reply burn-tch/build.rs:2]: OK. Not sure why there is no space.

**Comment by antimora** [comment]
Fixed the issues. Currently the CI is checking.

**Pull request merged**